src/utilities/scroll-utils.test.ts

```diff
@@ -36,6 +36,37 @@ describe("ScrollUtils", () => {
             // Assert
             expect(getElementByIdMock).toBeCalledTimes(50);
         });
+
+        test("when element is not found after 50 attempts, it logs a console warning for development environment", () => {
+            //Arrange
+            const elementId = faker.random.uuid();
+            process.env.NODE_ENV = "development";
+            const consoleWarnMock = jest.spyOn(console, "warn");
+
+            // Act
+            jest.useFakeTimers();
+            ScrollUtils.scrollToElementById(elementId);
+            jest.runAllTimers();
+
+            // Assert
+            expect(consoleWarnMock).toBeCalled();
+        });
+
+        test("when scrollOption has initial delay, it calls setTimeout with supplied delay", () => {
+            //Arrange
+            const options = { initialDelay: 200 };
```

Comment:
Suggestion: Consider using `faker.random.number(min: __, max: __)` (you can set the min and max).